template/template.go

```diff
@@ -325,6 +324,9 @@ func (t *Template) Data(recv string, groupLabels model.LabelSet, alerts ...*type
 					delete(commonAnnotations, an)
 				}
 			}
+			if len(commonLabels) == 0 && len(commonAnnotations) == 0 {
```

Comment:
Why not move this below `for _, a := range alerts[1:] {`? Am I missing something?

Reply:
Yes it works too.